test/ajv-validator.spec.ts

```diff
@@ -0,0 +1,50 @@
+import * as sinon from 'sinon';
+import Ajv from 'ajv';
+import { expect } from 'chai';
+import { AjvValidator } from '../lib';
+
+describe('AjvValidator', () => {
+  let ajvValidator: AjvValidator;
+
+  beforeEach(() => {
+    ajvValidator = new AjvValidator();
+  });
+
+  afterEach(() => {
+    sinon.restore();
+  });
+
+  it('should be success result if schema not exist', async () => {
+    const fakeGetSchema = sinon.fake.returns(false);
+
+    sinon.replace(Ajv.prototype, 'getSchema', fakeGetSchema);
```

Comment:
По-моему вот так лучше читается и меньше пишется.

```suggestion
    const stubGetSchema = sinon.stub(Ajv.prototype, <any>'getSchema').returns(false);
```

только чекни синтаксис! Мог ошибиться